lib/services/awsAdapter.js

```diff
@@ -44,6 +44,35 @@ function AWSAdapter (ec2) {
 			return result;
 		});
 	});
+
+	this.terminateInstances = Bluebird.method(function (instanceId) {
```

Comment:
Is this `Bluebird.method()` call necessary? It looks like everything in this method is already in a promise chain.